pom.xml

```diff
@@ -29,10 +29,10 @@
       <email>vincent@latombe.net</email>
     </developer>
   </developers>
-  <scm>
-    <connection>scm:git:git://github.com/jenkinsci/aws-java-sdk-plugin.git</connection>
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
+    <connection>scm:git:https://github.com/jenkinsci/aws-java-sdk-plugin.git</connection>
```

Comment:
change to https to fix
`[WARNING] <connection>scm:git:git://github.com/jenkinsci/aws-java-sdk-plugin.git/aws-java-sdk-ec2</connection> is invalid because git:// URLs are deprecated. Replace it with <connection>scm:git:https://github.com/jenkinsci/aws-java-sdk-plugin.git/aws-java-sdk-ec2</connection>. In the future this warning will be changed to an error and will break the build.`